management/global/organizations/aws-tf-organization.md

```diff
@@ -0,0 +1,99 @@
+
+### Create the initial directory structure for a new account
+As an example, we will set up the `apps-prd` account by using the `apps-devstg` as a source reference code:
+1. Ensure you are at the root of this repository
+2. Create the initial directory structure for the new account:
+    ```
+    mkdir -p apps-prd/global
+    mkdir -p apps-prd/us-east-1
+    ```
+3. Set up the config files:
+    1. Create a config files for this account: `cp -r apps-devstg/config apps-prd/config`
+    2. Open `apps-prd/config/backend.tfvars` and replace any occurrences of `devstg` with `prd`
+    3. Do the same with `apps-prd/config/account.tfvars`
+    4. Open up `apps-prd/config/backend.tfvars` again and replace this:
+        ```
+        profile = "bb-apps-prd-devops"
+        ```
+        with this:
+        ```
+        profile = "bb-apps-prd-oaar"
+        ```
+    5. In the step above, we are switching to the OAAR (OrganizationalAccountAccessRole) role because we are working with a brand new account that is empty, so, the only way to access it programmatically is through the OAAR role.
+    6. Now it's time to configure your OAAR credentials (if haven't already done so). For that you can follow the steps in [this section](https://leverage.binbash.com.ar/first-steps/management-account/#update-the-bootstrap-credentials) of the official documentation.
+4. Create the `base-tf-backend` layer:
+    1. Copy the layer from an existing one: `cp -r apps-devstg/us-east-1/base-tf-backend apps-prd/us-east-1/base-tf-backend`
+    2. Go to the `apps-prd/us-east-1/base-tf-backend` directory and open the `config.tf` file. Comment this block:
+        ```
+        backend "s3" {
+        key = "apps-devstg/tf-backend/terraform.tfstate"
+        }
+        ```
+    3. Now run the [Terraform workflow](https://leverage.binbash.com.ar/user-guide/ref-architecture-aws/workflow/) to initialize and
+       apply this layer.  (You may need to pass the `--skip-validation` flag to
+       `leverage tf init`.)
+    4. Open the `config.tf` file again and un-comment the block you commented before but first make sure you replace any occurrences of `devstg` with `prd`
+    5. Now run `leverage tf init`. Terraform should detect that you are trying to move a local state to a remote state and should prompt you for confirmation.
+5. Before moving on, go back to the root of this repository
+6. Now let's set up the base identities for the new account:
+    1. Create this layer from an existing one: `cp -r apps-devstg/global/base-identities apps-prd/global/base-identities`
+    2. Go to the `apps-prd/global/base-identities` directory and open the `config.tf` file. Replace any occurrences of `devstg` with `prd`
+    3. Now run `leverage tf init`
+    4. Import the OAAR role: `leverage terraform import module.iam_assumable_role_oaar.aws_iam_role.this OrganizationAccountAccessRole`
+    5. Now run `leverage tf apply`
+
+7. It's time for add a `security-base`  
```

Comment:
Just adding the colon at the end? (to keep the document style)

Reply:
@juanmatias updated thanks!